packages/manager/apps/pci-cold-archive/src/pages/containers/new/steps/LinkUserSelector.component.tsx

```diff
@@ -107,7 +107,7 @@ export default function LinkUserSelector({
 
         <div className="flex gap-4 items-center">
           <OdsSelect
-            className="min-w-[35rem]"
+            className="w-full max-w-[35rem]"
```

Comment:
It's still working on desktop? No need to specify `md:min-w-[35rem]` to keep the same behavior on desktop?